### ISSUE TITLE
Hover over graph functionality gone

### ISSUE THREAD

**Issue by BertSchel** [algenty/grafana-flowcharting#307]
First of all thanks for the great plugin, been using it for a long time now and very satisfied with it!
Some grafana-versions back there was a functionality that allowed you to hover over a grafana graph and show the corresponding values of the ruler in the flowchart next to it. For me this is not working anymore, is there something I can do on my end to get it working again?
There is a video of the functionality I mean in issue #242 posted by a user called "bijwaard". [This](https://github.com/algenty/grafana-flowcharting/files/5566854/vokoscreen-2020-11-19_14-00-15.mkv.zip) is the videolink.

**Comment by bijwaard** [comment]
Hi BertSchel,
Sorry, I didn't see your message, I wasn't referenced with the @ sign...
The tooltips are still working for me, even in grafana 8.1.2. There are some other issues with the newest grafana however, as mentioned in issue #314.
![image](https://user-images.githubusercontent.com/6523124/131320853-60583967-17a6-4397-b21b-a9e83c1164b4.png)
Kind regards,
Dennis

**Comment by bijwaard** [comment]
Sorry, I understood it wrong, you probably mean the ctrl-O hoover behavior that updates the values in the flowchart-time-aligned with the selection in the graph panels. That indeed seems to be broken in grafana 8.1.1 and 8.1.2 that I've just checked.

**Comment by BertSchel** [comment]
Yes that's what i mean indeed! Will there be a fix for this or Will it stay like this? Thx

**Comment by bijwaard** [comment]
Hi Bert,
Arnaud seems to lay low at the moment, since his career switch around April (https://github.com/algenty/grafana-flowcharting/issues/290#issuecomment-827342286). Hope he soon finds energy again for his fantastic plugin.
Kind regards,
Dennis

**Comment by BertSchel** [comment]
> Hi Bert,
> Arnaud seems to lay low at the moment, since his career switch around April (https://github.com/algenty/grafana-flowcharting/issues/290#issuecomment-827342286). Hope he soon finds energy again for his fantastic plugin.
> Kind regards,
> Dennis

Thx for the quick response! Great plugin indeed.
